example_project/measures/BuildResidentialModel/measure.rb

```diff
@@ -7,7 +7,7 @@
 # http://nrel.github.io/OpenStudio-user-documentation/measures/measure_writing_guide/
 
 require 'openstudio'
-resources_path = File.absolute_path(File.join(File.dirname(__FILE__), '../../resources/hpxml-measures/HPXMLtoOpenStudio/resources'))
+resources_path = File.absolute_path(File.join(File.dirname(__FILE__), '../../resources/resstock/resources/hpxml-measures/HPXMLtoOpenStudio/resources'))
```

Comment:
Yes, we're now using files from a resource folder within a resource folder within a resource folder...